app/models/spree/order_decorator.rb

```diff
@@ -21,4 +21,24 @@
         # .where.not(email: nil)
   }
 
-end
\ No newline at end of file
+  state_machine do
+    after_transition to: :complete, do: :send_risk_notification, if: :is_risky?
+  end
+
+  private
+
+  def send_risk_notification
+    SpreeSam::Notifications.build(:parse,
```

Comment:
is :parse the backend? if so, what if it's moved to the configurator so you can switch easily between pusher services